client/post-editor/media-modal/detail/detail-preview-videopress.jsx

```diff
@@ -102,4 +168,6 @@ class EditorMediaModalDetailPreviewVideoPress extends Component {
 	}
 }
 
-export default EditorMediaModalDetailPreviewVideoPress;
+export default connect(
+	null,
```

Comment:
is there any reason to pass this `null` here or to use `connect()` at all? looks like we could eliminate it